scriptHelper.js

```diff
@@ -1,43 +1,96 @@
-                    <li>Number of Moons: </li>
-                </ol>
-                <img src="">
-   */
+    // Here is the HTML formatting for our mission target div.
+    let missionTarget=document.getElementById("missionTarget")
+    missionTarget.innerHTML=`
+                    <h2>Mission Destination</h2>
+                    <ol>
+                        <li>Name: ${name} </li>
+                        <li>Diameter:${diameter} </li>
+                        <li>Star: ${star}</li>
+                        <li>Distance from Earth:${distance} </li>
+                        <li>Number of Moons: ${moons}</li>
+                    </ol>
+                    <img src="${imageUrl}">`
 }
 
 function validateInput(testInput) {
-   
+    if(testInput===""){
+        return "Empty"
+    } else if (isNaN(Number(testInput))){
+        return "Not a Number"
+    } else {
+        return "Is a Number"
+    }
 }
 
-function formSubmission(document, list, pilot, copilot, fuelLevel, cargoLevel) {
-   
+function formSubmission (document, pilot, copilot, fuelLevel, cargoMass) {
+    pilot = document.querySelector("input[name=pilotName]").value;
+    copilot = document.querySelector("input[name=copilotName]").value;
+    fuelLevel = document.querySelector("input[name=fuelLevel]").value;
+    cargoMass = document.querySelector("input[name=cargoMass]").value;
+    let pilotStatus=document.getElementById("pilotStatus");
+    let copilotStatus=document.getElementById("copilotStatus");
+    let fuelStatus=document.getElementById("fuelStatus");
+    let cargoStatus=document.getElementById("cargoStatus");
+    let faultyItems=document.getElementById("faultyItems");
+    let launchStatus=document.getElementById("launchStatus");
+
+    if (validateInput(pilot) === "Empty" || validateInput(copilot) === "Empty" || validateInput(fuelLevel) === "Empty" || validateInput(cargoMass) === "Empty") {
+        faultyItems.style.visibility = "hidden";
+        launchStatus.innerHTML = "Awaiting Information Before Launch.<br>Please check field inputs.";
+        launchStatus.style.color = "rgb(199, 37, 78)"
+        alert("All fields are required!");
+    } else if (validateInput(pilot) === "Is a Number" || validateInput(copilot) === "Is a Number" || validateInput(fuelLevel) === "Not a Number" || validateInput(cargoMass) === "Not a Number") {
+        faultyItems.style.visibility = "hidden";
+        launchStatus.innerHTML = "Awaiting Information Before Launch.<br>Please check field inputs.";
+        launchStatus.style.color = "rgb(199, 37, 78)"
+        alert("All fields need valid information.");
+    } else { 
+        faultyItems.style.visibility = "visible";
+        pilot.innerHTML = `Pilot ${pilot} is ready for launch`;
+        copilot.innerHTML = `Co-pilot ${copilot} is ready for launch`;
+        let launchStatus = document.getElementById("launchStatus");
+        if (Number(fuelLevel) < 10000 && Number(cargoMass) < 10000) {
```

Comment:
```suggestion
        if (Number(fuelLevel) < 10000 && Number(cargoMass) <= 10000) {
```
For `cargoMass`, it is acceptable if it is less than or equal to 10000, like `fuelLevel` is acceptable if greater than or equal to 10000.